pkg/apis/submariner.io/v1/types.go

```diff
@@ -78,18 +78,21 @@ type EndpointSpec struct {
 	// +kubebuilder:validation:MinLength=1
 	ClusterID string `json:"cluster_id"`
 	CableName string `json:"cable_name"`
+	// Deprecated: Use HealthCheckIPs
```

Comment:
Perhaps this should point to the getter/setter too:

```suggestion
	// Deprecated: Use Get/SetHealthCheckIP() or, if necessary, HealthCheckIPs
```